docs/CONTRIBUTING.md

```diff
@@ -185,6 +186,19 @@ experimental state. This means that while it may function, we are not intending
 to provide work-arounds for building on that platform while it is still in an
 unstable state.
 
+#### Bootstrapping build dependencies
```

Comment:
This section should be moved below "Getting Set Up (Rust)" as "Getting Set Up (JVM)". Content can stay the same. Also please check all docs for title case.